### ISSUE TITLE
[Suggestion] Boosty and Patreon integration

### ISSUE THREAD

**Issue by Dizover** [IAmSneak/diamond-economy#21]
Server admins could profit from Boosty and Patreon if a player would receive in-game currency from subscribing to the server admin. Also, make a separate in-game currency that would be issued for a subscription and the ability to configure the scale of converting real currency into in-game currency into a separate in-game wallet. The admins of the server could also subscribe to your Boosty or Patreon, because they would have money.

If there is a choice of payment between different types of currency in the Diamond Chest Shop, then the server admins could sell in-game items and it would be legal, I think.

But mostly I was thinking about the monthly profit from the territories that the players would own. 

In short, the user subscribes to B or P, a separate in-game currency is credited to the player in a separate in-game wallet, the player buys in-game items for one or another in-game currency in the Diamond Chest Shop. And I may be one of the developers of the fashion for the private territory will agree to the diamond-economy integration.

Of course, everything can break about the impossibility of the task, but without asking I will not know if it is possible.

**Comment by IAmSneak** [comment]
theoretically if a server wants currency to be bought with irl money it would just require the command to be run. The problem I see with that is that there is no way to send money to offline players, would would probably happen at least of the time with something like that.

Support for sending money to offline players would be nice.
